app/components/UI/Tooltip.js

```diff
@@ -0,0 +1,89 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { Box, Text } from 'rebass'
+
+const StyledTooltipIconBox = styled(Box)`
+  border-radius: 50%;
+  height: 13px;
+  line-height: 13px;
+  margin-left: 5px;
+  position: relative;
+  text-align: center;
+  transform: translateZ(0);
+  width: 13px;
+  z-index: 2;
+
+  &:hover {
+    cursor: pointer;
+  }
+`
+StyledTooltipIconBox.displayName = 'TooltipIconBox'
+
+const StyledTooltipWrapper = styled(Box)`
+  border: 1px solid white;
+  border-radius: 4px;
+  padding: 5px 5px 5px 25px;
+  position: absolute;
+  top: -5px;
+  width: 225px;
+  z-index: 1;
+  transform: translateZ(0);
```

Comment:
Is this related to https://stackoverflow.com/questions/10814178/css-performance-relative-to-translatez0 ?
If so then it seems like it's not a recommended approach

Reply:
No, this was to put it on its own rendering layer so that when I use z-index on the icon and the surrounding layer they are relative to each other. Without this trick it doesn't seem to work.

Reply:
Actually, just checked and it seems to work fine without the `translateZ`. Removed.